core/src/main/java/nu/yona/server/properties/SecurityProperties.java

```diff
@@ -5,8 +5,9 @@
 	private int newDeviceRequestExpirationDays = 1;
 	private int passwordLength = 32;
 	private long bruteForceBlockMinutes = 60;
-	private long dosMemorySeconds = 300;
-	private boolean isDOSProtectionEnabled = false;
+	private long dosProtectionWindowSeconds = 300;
+	private boolean isDosProtectionEnabled = false;
+	private int maxCreateUserAttemptsPerTimeWindow = 1;
```

Comment:
You updated the number to 2 in application.properties. I think 2 is OK, addUser may indeed be called twice in the resubscribe scenario (the first time it will give an exception and the app discovers that the user exists).